src/index.tsx

```diff
@@ -2322,19 +2322,58 @@ export class App extends React.Component<any, AppState> {
 const rootElement = document.getElementById("root");
 
 class TopErrorBoundary extends React.Component {
-  state = { hasError: false, stack: "", localStorage: "" };
+  state: {
+    unresolvedError: Error | null;
+    hasError: boolean;
+    stack: string;
+    localStorage: string;
+  } = {
+    unresolvedError: null,
```

Comment:
Could this be rewritten as

```suggestion
    unresolvedError: null as Error | null,
```

to avoid having to write out full types?

Reply:
Not gonna work for the commit I'm cooking.

Reply:
I need to also supply the type to the component class' generic otherwise callback-setState yells at me. If you can fix that, then we can use type assertion indeed.

Reply:
See https://github.com/excalidraw/excalidraw/pull/809/commits/e08a6b97315df724556554642eea940ec1fae25e#diff-38682f47b0cf12c516bc3714fc45d0fdR2348